website/docs/cloud-docs/api-docs/teams.mdx

```diff
@@ -146,14 +148,14 @@ This POST endpoint requires a JSON object with the following properties as a req
+-> **Note:** You cannot set `manage-workspaces` to `false` when setting `manage-projects` to `true`, since project permissions cascade down to workspaces. This is also the case for `read-workspaces` and `read-projects`, if `read-projects` is `true`, `read-workspaces` must be `true` as well.
 
 | Key path                                | Type   | Default    | Description                                                                                                                                                                                                                                                                                                       |
 |-----------------------------------------|--------|------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `data.type`                             | string |            | Must be `"teams"`.                                                                                                                                                                                                                                                                                                |
 | `data.attributes.name`                  | string |            | The name of the team, which can only include letters, numbers, `-`, and `_`. This will be used as an identifier and must be unique in the organization.                                                                                                                                                           |
 | `data.attributes.sso-team-id`           | string | (nothing)  | The unique identifier of the team from the SAML `MemberOf` attribute. Only available in Terraform Enterprise 202204-1 and later, or if the team belongs to an organization in the paid Terraform Cloud Business tier.                                                                                             |
-| `data.attributes.organization-access`   | object | (nothing)  | Settings for the team's organization access. This object can include `manage-policies`, `manage-policy-overrides`, `manage-run-tasks`, `manage-workspaces`, `manage-vcs-settings`, `manage-providers`, `manage-modules`, and `manage-projects` properties with boolean values. All properties default to `false`. |
+| `data.attributes.organization-access`   | object | (nothing)  | Settings for the team's organization access. This object can include `manage-policies`, `manage-policy-overrides`, `manage-run-tasks`, `manage-workspaces`, `manage-vcs-settings`, `manage-providers`, `manage-modules`, `manage-projects`, `read-projects`, and `read-workspaces` properties with boolean values. All properties default to `false`. |
```

Comment:
```suggestion
| `data.attributes.organization-access`   | object | (nothing)  | Settings for the team's organization access. This object can include the `manage-policies`, `manage-policy-overrides`, `manage-run-tasks`, `manage-workspaces`, `manage-vcs-settings`, `manage-providers`, `manage-modules`, `manage-projects`, `read-projects`, and `read-workspaces` properties with boolean values. All properties default to `false`. |
```

This list of properties is getting pretty long - we might think about a better way to handle it if we add too many more.